docs/kqueen.rst

```diff
@@ -152,26 +152,235 @@ in the configuration file, set the environment variable matching the KQUEEN_<con
 
-  # Move data to etcd directory
-  mv -v /root/backup/* /var/lib/etcd/default/
+.. code-block:: bash
+
+   TOKEN=$(curl -s -H "Content-Type: application/json" --data '{"username":"admin","password":"default"}' -X POST <<kqueen_api_host>>:5000/api/v1/auth | jq -r '.access_token'); echo $TOKEN
+   curl -H "Authorization: Bearer $TOKEN" <<kqueen_api_host>>:5000/metrics/
+
+
+* Prometheus API:
+
+   1. Add the scraper IP address to PROMETHEUS_WHITELIST configuration.
+   2. Run the following command:
+
+      .. code-block:: bash
+
+         curl <<prometheus_host>>:<<prometheus_port>>/metrics
+
+
+All application metrics are exported to the **/metrics**  API endpoint. Any external Prometheus instance can then scrape this metric.
+
+
+Provision a Kubernetes cluster
+------------------------------
+
+You can provision a Kubernetes cluster using various community of engines, such as Google Kubernetes engine or Azure Kubernetes Service.
+
+
+To provision a Kubernetes cluster using the Google Kubernetes Engine:
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+1. Login in the Google Kubernetes Engine (https://console.cloud.google.com).
+2. Select your Project.
+3. Navigate to the ``API’s & Services``  -> ``Credentials`` tab and click ``Create credentials``.
+4. From ``Service Account key``, select your service account.
+5. Select Json as the key format.
+6. Download the Json snippet.
+7. Log in to the KQueen web UI.
+8. From the ``Create Provisioner`` tab, select ``Google Kubernetes Engine``.
+9. Specify your project ID (``Project info`` tab on the main page of the GCE Dashboard https://console.cloud.google.com).
+10. Insert the downloaded Json snippet that contains the service account key and submit the provisioner creation.
+11. Click ``Deploy Cluster``.
+12. Select the defined GCE provisioner.
+13. Specify the cluster requirements.
+14. Click ``Submit``.
+15. To track the cluster status, navigate to the KQueen main dashboard.
+
+To provision a Kubernetes cluster using the Azure Kubernetes Service:
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+1. Log in to https://portal.azure.com.
+2. Create an Azure Active Directory Application as described in the official Microsoft `Documentation <https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-group-create-service-principal-portal#create-an-azure-active-directory-application>`_.
+3. Copy the Application ID, Application Secret, Tenant ID (Directory ID), and Subscription ID to use in step 8.
+4. Set the ``Owner`` role to your Application in the Subscription settings to enable the creation of Kubernetes clusters. For details, see the latest steps in the Microsoft official documentation described in step 2. Manually save the Application secret because you are not able to retrieve the secret later. You provide the secret value with the application ID to log in as the application. Store the key value where your application can retrieve it.
```

Comment:
Manually save the Application secret since it is not possible to retrieve the secret later. The secret value and the application ID are required for the application authorization? Store the key value where your application can retrieve it.

(please check if the second sentence is technically correct)

see the [which steps exactly?] latest steps in the Microsoft official documentation. - please add a link instead of pointing to step 2.

Reply:
removed, coz this info already described into docs microsoft, so imo its redundant